test/test_jit.py

```diff
@@ -11877,18 +11874,23 @@ def test_lstm_cuda(self):
         self.assertGraphContainsExactly(
             forward_graph, 'prim::FusionGroup', 1, consider_subgraphs=True)
         self.assertExpectedGraph(forward_graph, subname='forward')
+        self.assertTrue(len(forward_graph.nodes()) == 2)
+        FileCheck().check("DifferentiableGraph").check_next("TupleConstruct") \
+            .check_next("return").run(str(forward_graph))
 
         hy, cy = module(*inputs)
         (hy + cy).sum().backward()
-        self.assertExpectedGraph(backward_graph(module), subname='backward')
+        backward = backward_graph(module)
+        FileCheck().check("FusionGroup_0").check_next("FusionGroup_1").run(str(backward))
```

Comment:
Can we check that there are no more than 2 fusion groups in the backward? The fewer the merrier